### PR TITLE
[7/7] Refactor network - The rest 

### PR DESCRIPTION
**Pull request by nikkolasg**
This PR shows the rest of the changed code in order to adapt for this new network stack. This can be quite big but there's no big structural changes here.

* `sda.Host` renamed to `sda.Conode` :)

### CODE REVIEW

File: app/cothorityd/cothorityd.go [hunk 1]
```diff
@@ -110,10 +110,7 @@ func runServer(ctx *cli.Context) {
 	if err != nil {
 		log.Fatal("Couldn't parse config:", err)
 	}
-	host.ListenAndBind()
-	host.StartProcessMessages()
-	host.WaitForClose()
-
+	host.Start()
```
Comment (ineiti) [inline app/cothorityd/cothorityd.go]: Very nice. But should be called `conode` now ;)

**Comment by ineiti** [comment]
This branch should have `refactor_network_identity` merged in.

File: protocols/bftcosi/bftcosi_test.go [hunk 2]
```diff
@@ -245,6 +247,7 @@ func runProtocolOnceGo(nbrHosts int, name string, refuseCount int,
 		log.Lvl1("Going to break because of timeout")
 		return errors.New("Waited " + wait.String() + " for BFTCoSi to finish ...")
 	}
+	log.Print("out of runProtocolOnceGo")
```
Comment (Liamsi) [inline protocols/bftcosi/bftcosi_test.go]: This sounds like you want to remove this output or not?

File: services/services.go [hunk 3]
```diff
@@ -3,7 +3,9 @@ package services
 import (
 	// Importing the services so they register their services to SDA
 	// automatically when importing github.com/dedis/cothority/services
-	_ "github.com/dedis/cosi/service"
+	// XXX We should not import that doing weird cross dependancy between
+	// packages
+	//	_ "github.com/dedis/cosi/service"
```
Comment (Liamsi) [inline services/services.go]: Not sure what this comment means. If it's legitimate to remove `_ "github.com/dedis/cosi/service"`, you can just do it, or not?

**Comment by ineiti** [comment]
Only @Liamsi comment is missing...

**Comment by ineiti** [comment]
BIG nitpick: all occurrences of variables called `host` or `h` should be renamed `conode` or `c`.

**Comment by nikkolasg** [comment]
Done the renaming to conode (with the nitpick part where I could) + ismail comments.

**Comment by nikkolasg** [comment]
Just as a side note: look at how many comments there is in this PR (due to high number of files changed) vs the firsts ones... :D

**Pull request merged**